src/services/member/schemas.ts

```diff
@@ -6,7 +6,25 @@ import {
 
 import { NAME_REGEX, UUID_REGEX } from '../../schemas/global';
 
-const EMAIL_REGEX = '^[\\w-\\.]+@([\\w-]+\\.)+[\\w-]{2,4}$';
+/**
+ * This allows email adresses that are structured as follows:
+ * - at least one of
+ *   - word character
+ *   - dash
+ *   - dot
+ * - the @ symbol
+ * - multiple times
+ *   - at least one of
+ *     - word cahracter
+ *     - dash
+ *   - a dot
+ * - a tld extension that is between 2 and 63 chars long
+ *
+ * The maximum length of the TLD is defined by https://www.rfc-editor.org/rfc/rfc1034
+ * and is 63 octets (chars)
+ *
+ */
+export const EMAIL_REGEX = '^[.-\\w]+@[.-\\w]+.[-\\w]{2,63}$';
```

Comment:
Thanks for the changes, but do we really need this regex? I let you decide if it's better to keep the regex, or if we simplify the schema.

Reply:
@pyphilia I see, well, this regex might be a bit strict, so we would be better off in the future to remove it. I do not think this is too urgent, and I am proud of what I did for the tests ahah.